packages/core/src/theme/Theme.ts

```diff
@@ -13,7 +13,8 @@ export type characteristic =
   | "status"
   | "taggable"
   | "target"
-  | "text";
+  | "text"
+  | "differential" /* **deprecated** */;
```

Comment:
needs a change in `ColorChooser.css`

Reply:
and `MetricContent.css`

Reply:
also missing in `tooling/stylelint/custom-property-no-foundations/index.js`

Reply:
it's not in the first 2 files, i think you may be looking on another branch comparing? fixed last one